[debug] [2019-03-05T03:46:34.395Z] ----------------------------------------------------------------------
[debug] [2019-03-05T03:46:34.397Z] Command:       /home/arenglish/.nvm/versions/node/v10.15.0/bin/node /home/arenglish/.nvm/versions/node/v10.15.0/bin/firebase deploy
[debug] [2019-03-05T03:46:34.398Z] CLI Version:   6.3.1
[debug] [2019-03-05T03:46:34.398Z] Platform:      linux
[debug] [2019-03-05T03:46:34.398Z] Node Version:  v10.15.0
[debug] [2019-03-05T03:46:34.398Z] Time:          Mon Mar 04 2019 21:46:34 GMT-0600 (Central Standard Time)
[debug] [2019-03-05T03:46:34.398Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-03-05T03:46:34.406Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-03-05T03:46:34.407Z] > authorizing via signed-in user
[debug] [2019-03-05T03:46:34.407Z] [iam] checking project portfolio-dcde8 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2019-03-05T03:46:34.408Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/portfolio-dcde8:testIamPermissions  

[debug] [2019-03-05T03:46:34.750Z] <<< HTTP RESPONSE 200
[debug] [2019-03-05T03:46:34.753Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/portfolio-dcde8  
 
[debug] [2019-03-05T03:46:35.103Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'portfolio-dcde8'...
[info] 
[info] i  deploying hosting
[debug] [2019-03-05T03:46:35.116Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.resolve (path.js:1085:7)
    at module.exports (/home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/lib/resolveProjectPath.js:5:17)
    at /home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/lib/deploy/hosting/prepare.js:46:36
    at arrayEach (/home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:516:11)
    at Function.forEach (/home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9344:14)
    at module.exports (/home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/lib/deploy/hosting/prepare.js:29:7)
    at _chain (/home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/lib/deploy/index.js:22:40)
    at /home/arenglish/.nvm/versions/node/v10.15.0/lib/node_modules/firebase-tools/lib/deploy/index.js:63:16
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
